environment.yml

```diff
@@ -23,11 +23,13 @@ dependencies:
     - isort[colors]>=5.8
     - jupyterlab
     - kornia>=0.5.4
+    - laspy
```

Comment:
Any interest in trying to create a Spack package for this? If not I can do it.

Reply:
I'm working on creating dummy test data. If you could add it that would be great.

Reply:
Do you need laszip or lasrs support?

Reply:
I don't believe so. The files are `.las` and I installed with no extras so the only dependency I'm using for laspy is numpy.